README.md

```diff
@@ -218,6 +218,36 @@ To target a specific test you can run `python runtests.py tests.test_file.TheTes
 
 Tests are written against Wagtail 2.10 and later.
 
+### Customizing the save method
+In some cases you may want to customize how saving works, like making the save to airtable asynchronous for example.
+
+To do so, set: WAGTAIL_AIRTABLE_SAVE_SYNC=False in your settings.py file. 
+
+This _escapes_ out of the original save method and requires you enable the asynchronous part of this on your own.
+
+An example of how you might set this up using the wagtail_hook `after_page_publish` action using [django_rq](https://github.com/rq/django-rq)
+```
+#settings.py
+WAGTAIL_AIRTABLE_SAVE_SYNC=False
+WAGTAIL_AIRTABLE_PUSH_MESSAGE="Airtable save happening in background"
+
+#wagtail_hooks.py
+@job('airtable')
+def async_airtable_save(page_id):
+    my_page = Page.objects.get(page_id)
```

Comment:
This is missing a `specific`, so I think would fail as the `save_to_airtable` isn't on the base `Page` model